### DISCUSSION TITLE
issues with legacy versions of libraries in libman

### DISCUSSION THREAD

**Discussion by the-doctor1994** [cdnjs/cdnjs#14143] [General] [answered]
our company is having issues building our applications that use libman today, when we link to a library like below there is an error unless we change the version from 2.8.0 to 3.8.0 (the newest version, all other versions do not work). is there a known issue with the CDNJS service? or a workaround? have all older versions of this library been removed from the CDN?
`"version": "1.0",
  "defaultProvider": "cdnjs",
  "libraries": [
    {
      "library": "Chart.js@2.8.0",
      "destination": "wwwroot/lib/chart.js"
    },`

**Comment by MattIPv4** [comment, marked as answer]
Please see #14141 and https://github.com/aspnet/LibraryManager/issues/685 -- LibMan is relying on a deprecated part of the cdnjs API to operate, which is currently phased out for 10% of requests.
